avm/res/dev-ops-infrastructure/pool/main.bicep

```diff
@@ -0,0 +1,506 @@
+    }
+    scope: managedDevOpsPool
+  }
+]
+
+@description('The name of the Managed DevOps Pool.')
+output name string = managedDevOpsPool.name
+
+@description('The resource ID of the Managed DevOps Pool.')
+output resourceId string = managedDevOpsPool.id
+
+@description('The name of the resource group the Managed DevOps Pool resource was deployed into.')
+output resourceGroupName string = resourceGroup().name
+
+@description('The location the Managed DevOps Pool resource was deployed into.')
+output location string = managedDevOpsPool.location
+
+@description('The principal ID of the system assigned identity.')
+output systemAssignedMIPrincipalId string? = managedDevOpsPool.?identity.?principalId
+
+type osProfileType = {
+  @description('Required. The logon type of the machine.')
+  logonType: ('Interactive' | 'Service')
+
+  @description('Optional. The secret management settings of the machines in the pool.')
+  secretsManagementSettings: {
+    @description('Required. The secret management settings of the machines in the pool.')
+    keyExportable: bool
+
+    @description('Required. The list of certificates to install on all machines in the pool.')
+    observedCertificates: string[]
+
+    @description('Optional. Where to store certificates on the machine.')
+    certificateStoreLocation: string?
+  }?
+}
+
+type storageProfileType = {
+  @description('Optional. The Azure SKU name of the machines in the pool.')
+  osDiskStorageAccountType: ('Premium' | 'StandardSSD' | 'Standard')?
+
+  @description('Optional. A list of empty data disks to attach.')
+  dataDisks: {
+    @description('Optional. The type of caching to be enabled for the data disks. The default value for caching is readwrite. For information about the caching options see: https://blogs.msdn.microsoft.com/windowsazurestorage/2012/06/27/exploring-windows-azure-drives-disks-and-images/.')
+    caching: ('None' | 'ReadOnly' | 'ReadWrite')?
+
+    @description('Optional. The initial disk size in gigabytes.')
+    diskSizeGiB: int?
+
+    @description('Optional. The drive letter for the empty data disk. If not specified, it will be the first available letter. Letters A, C, D, and E are not allowed.')
+    driveLetter: string?
+
+    @description('Optional. The storage Account type to be used for the data disk. If omitted, the default is Standard_LRS.')
+    storageAccountType: ('Premium_LRS' | 'Premium_ZRS' | 'StandardSSD_LRS' | 'StandardSSD_ZRS' | 'Standard_LRS')?
+  }[]?
+}?
+
+type imageType = {
+  @description('Optional. List of aliases to reference the image by.')
+  aliases: string[]?
+
+  @description('Optional. The percentage of the buffer to be allocated to this image.')
+  buffer: string?
+
+  @description('Required. The image to use from a well-known set of images made available to customers.')
+  wellKnownImageName: string
+
+  @description('Optional. The resource id of the image.')
+  resourceId: string?
+}[]
+
+type organizationGitHubType = {
```

Comment:
Review to myself: Got a review from a community member. GitHub won't be supported, so this can be removed.

Reply:
Fixed.

Reply:
Can be resolved @johnlokerse ?

Reply:
Yes, GitHub is removed. Resolved!